package.json

```diff
@@ -8,7 +8,7 @@
     "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2020 browser module main",
     "prepare": "git config core.hookspath .githooks",
     "start": "nx serve",
-    "build": "nx run-many --target build --all",
+    "build": "nx run-many --target build --all --exclude=astro-app",
```

Comment:
Do we need to exclude this?

Reply:
We do not need to exclude it but it also doesn't bring any value to the CI (assuming e2e for it is in place)

Reply:
👍